apps/docs/src/pages/development.tsx

```diff
@@ -135,6 +135,21 @@ const Development = () => {
             <img src="/img/logo_uzh.jpeg" className="h-12" />
           </div>
         </div>
+        <div className="flex flex-col items-center">
```

Comment:
Maybe it would make sense to either move this video into a separate section with a short description (instead of "video explanation" below it) or somehow change the layout a bit to avoid that the roadmap is separated from its description